mne/gui/_backend.py

```diff
@@ -25,3 +27,8 @@ def _check_backend():
                "recommend using qt4 which can be enabled by installing the "
                "pyside package." % backend)
     warning(None, msg, "Pyface Backend Warning")
+
+
+def _testing_mode():
+    """Helper to determine if we're running tests."""
```

Comment:
FYI the suggested standard is to us imperative tone, so `"""Determine if we're..."""` is preferred (but is a non-essential change). "Helper" is implied anyway by the private function-ness